Woodoku/tests/entity/test_woodoku_board.py

```diff
@@ -0,0 +1,163 @@
+from typing import Tuple
+
+import numpy as np
+import pytest
+
+from exceptions.exceptions import ShapeOutOfBoardError
+from woodoku_board import WoodokuBoard
+from woodoku_shape import WoodokuShape
+
+
+class TestWoodokuBoard:
+    l_shape: WoodokuShape = WoodokuShape([(0, 0), (1, 0), (1, 1), (1, 2)])
+    horizontal_bar_shape: WoodokuShape = WoodokuShape([(0, 0), (0, 1), (0, 2), (0, 3), (0, 4)])
+    gun_shape: WoodokuShape = WoodokuShape([(0, 0), (0, 1), (0, 2), (1, 0)])
+    vertical_two_block: WoodokuShape = WoodokuShape([(0, 0), (1, 0)])
+    one_block: WoodokuShape = WoodokuShape([(0, 0)])
+
+    @pytest.mark.parametrize('shape', [
+        l_shape,
+        horizontal_bar_shape,
+        gun_shape
+    ])
```

Comment:
Since they are simple and common shapes, I add them to test the most basic addition operation. We could add more complex shapes if they are not complex enough